src/main/java/com/sngular/kloadgen/extractor/extractors/JsonExtractor.java

```diff
@@ -19,16 +18,33 @@
 import com.sngular.kloadgen.model.json.ObjectField;
 import com.sngular.kloadgen.model.json.Schema;
 import com.sngular.kloadgen.model.json.StringField;
+import io.confluent.kafka.schemaregistry.ParsedSchema;
+import io.confluent.kafka.schemaregistry.json.JsonSchema;
 import org.apache.commons.collections4.CollectionUtils;
 import org.apache.commons.collections4.Transformer;
 import org.apache.commons.lang3.StringUtils;
 
-public class JsonExtractor {
+public class JsonExtractor implements Extractor {
 
   private final JSONSchemaParser jsonSchemaParser = new JSONSchemaParser();
 
-  public final List<FieldValueMapping> processSchema(final JsonNode jsonNode) {
-    return processSchema(jsonSchemaParser.parse(jsonNode));
+  @Override
+  public final List<FieldValueMapping> processApicurioParsedSchema(final Object schema) {
+    return processSchema(((io.apicurio.registry.serde.jsonschema.JsonSchema) schema).toJsonNode());
+  }
+
+  @Override
+  public final List<FieldValueMapping> processConfluentParsedSchema(final Object schema) {
+    return processSchema(((JsonSchema) schema).toJsonNode());
+  }
+
+  @Override
+  public final ParsedSchema getParsedSchema(final String schema) {
+    return new JsonSchema(schema);
+  }
+
```

Comment:
Please check the configuration of your IDE, for some reason it doesn't put the `Override` annotation on the last method in the extractor implementation